src/record/record.js

```diff
@@ -237,9 +240,10 @@ Record.prototype.discard = function() {
 	this.whenReady( function() {
 		this.usages--;
 		if( this.usages <= 0 ) {
-				this.emit( 'destroyPending' );
-				this._discardTimeout = setTimeout( this._onTimeout.bind( this, C.EVENT.ACK_TIMEOUT ), this._options.subscriptionTimeout );
-				this._connection.sendMsg( C.TOPIC.RECORD, C.ACTIONS.UNSUBSCRIBE, [ this.name ] );
+			this.isReady = false;
```

Comment:
Setting isReady to false allows the record to be revived with pending operations.